npm-shrinkwrap.json

```diff
@@ -2,6 +2,18 @@
   "name": "etcher",
   "version": "1.0.0-beta.19",
   "dependencies": {
+    "ajv": {
```

Comment:
Hmmm, so what were we using previously that had so many fewer dependencies? Is there something lighter-weight we could use than `request`?
Or should I stop worrying and learn to love code-bloat and npm dependency hell? ;-D